specs/agents/mobile/metrics.md

```diff
@@ -0,0 +1,41 @@
+## Mobile Metrics
+
+### CPU metrics
+| Name               | Type             | Units      | Description                     |
+|--------------------|------------------|------------|---------------------------------|
+| `system.cpu.usage` | Observable Guage | percentage | A percentage value of cpu usage |
+
+### Memory Metrics
+| Name                   | Type             | Units | Description                             |
+|------------------------|------------------|-------|-----------------------------------------|
+| `system.memory.usage`  | Observable Guage | bytes | The application's memory usage in bytes |
+
+
+### Application Metrics
+#### load times
+| Name                                          | Type      | Units   | Description                                                           |
+|-----------------------------------------------|-----------|---------|-----------------------------------------------------------------------|
+| `application.launch.optimizedTimeToFirstDraw` | histogram | seconds | The amount of time spent launching the app until loaded (optimized)   |
+| `application.launch.timeToFirstDraw`          | histogram | seconds | The amound of time spent launching the app until loaded               |
+| `application.launch.resumeTime`               | histogram | seconds | the amount of time spent resuming the application from the background | 
+
+#### responsiveness
+| Name                                   | Type      | Units   | Description                                                 |
+|----------------------------------------|-----------|---------|-------------------------------------------------------------|
+|  `application.responsiveness.hangtime` | histogram | seconds | The amount of time the applications has spent unresponsive. | 
+ 
+### Application exit
+Traces application exit counts in both healthy and unhealthy (crashes) states
+
+| Name               | Type  | Units | Description                   |
+|--------------------|-------|-------|-------------------------------|
+| `application.exit` | count |  unit | A count of application exits. |
```

Comment:
Maybe `application.exitCount` would be a better identifier.

Reply:
Is it common practice to add the type to the name of the metric? should we add the type to all the metric names?

Reply:
Not sure tbh if there is something like a best/common practice here.
For me, the first thought I had with the original suggestion was that it referred to the exit code of the application/process.
`exitCount` conveys the meaning better IMHO.